services/viva-ms/helpers/dynamoDb.js

```diff
@@ -0,0 +1,40 @@
+import * as dynamoDb from '../../../libs/dynamoDb';
+import config from '../../../config';
+import { CASE_HTML_GENERATED } from '../../../libs/constants';
+
+export async function getClosedUserCases(primaryKey) {
```

Comment:
No need to specify `async` because you don't have any `await` expression in the function body

Reply:
Done